src/session.rs

```diff
@@ -209,6 +209,26 @@ impl SessionBuilder {
 		ortsys![unsafe EnableProfiling(self.session_options_ptr, profiling_file.as_ptr()) -> OrtError::CreateSessionOptions];
 		Ok(self)
 	}
+	/// Enables/disables memory pattern optimization. Disable it if the input size varies, i.e., dynamic batch
+	pub fn with_memory_pattern(self, enable: bool) -> OrtResult<Self> {
+		if enable {
+			ortsys![unsafe EnableMemPattern(self.session_options_ptr) -> OrtError::CreateSessionOptions];
+		}
+		else {
+			ortsys![unsafe DisableMemPattern(self.session_options_ptr) -> OrtError::CreateSessionOptions];
+		}
+		Ok(self)
+	}
+	/// Enables/disables cpu mem areana allocator
+	pub fn with_cpu_memory_arena(self, enable: bool) -> OrtResult<Self> {
```

Comment:
Is this different than passing use_arena: true in `OrtSessionOptionsAppendExecutionProvider_CPU`? There's already `ExecutionProvider::with_use_arena`

Reply:
looks like the same. I'll remove this api